Improve style of 'no annotations' watermark

mwoehlke-kitware opened the pull request:
Use custom style rules instead of bootstrap for the 'no annotations' watermark. This allows us to fully tuck the watermark into the corner (without a funky notch due to corner rounding) and to give it a translucent background and border color that is aesthetically pleasing on top of video.

matthewma7 commented:
I think we could remove the `alert alert-danger` classes off the message div with this change.

mwoehlke-kitware committed:
Improve style of 'no annotations' watermark

Use custom style rules instead of bootstrap for the 'no annotations'
watermark. This allows us to fully tuck the watermark into the corner
(without a funky notch due to corner rounding) and to give it a
translucent background and border color that is aesthetically pleasing
on top of video.

matthewma7 reviewed (approved):
👌
